src/Symfony/Component/Clock/ClockInterface.php

```diff
@@ -11,10 +11,12 @@
 
 namespace Symfony\Component\Clock;
 
+use Psr\Clock\ClockInterface as PsrClockInterface;
+
 /**
  * @author Nicolas Grekas <p@tchwork.com>
  */
-interface ClockInterface
+interface ClockInterface extends PsrClockInterface
 {
     public function now(): \DateTimeImmutable;
```

Comment:
this method should be removed as it overrides the parent one removing the phpdoc now.

Reply:
updated!